src/git/actions/worktree.ts

```diff
@@ -56,11 +56,15 @@ export function copyChangesToWorktree(
 	});
 }
 
-export function open(worktree: GitWorktree, options?: { location?: OpenWorkspaceLocation; openOnly?: boolean }) {
+export function open(
+	worktree: GitWorktree,
+	options?: { location?: OpenWorkspaceLocation; openOnly?: boolean; prId?: string },
+) {
 	return executeGitCommand({
 		command: 'worktree',
 		state: {
 			subcommand: 'open',
+			prId: options?.prId,
```

Comment:
This is set here, but not sure where it is used in the command flow (see prev. comment)

Reply:
removed